### PR TITLE
New i18n

### LINKED ISSUE #1901: New translations mechanism
**Description**:
The current i18n implementation causes slower compilation for projects that have a lot of languages.
The translations do not work in development.
The i18n plugin is not verbose at all - does not provide any additional information about unused/missing translations.

**Additional context**:
ScandiPWA 3.2.4

### PR DESCRIPTION
**Pull request by yeegor**
Enable the new i18n mechanism, resolves #1901 
The PR has the following linked functionality:
1. https://github.com/scandipwa/locale/pull/5 to have full locale code on `window` from ScandiPWA
2. https://github.com/scandipwa/create-scandipwa-app/pull/6 the functionality itself

### CODE REVIEW

File: packages/scandipwa/package.json [hunk 1]
```diff
@@ -32,7 +32,8 @@
         "themeAlias": "Source",
         "type": "theme",
         "extensions": {
-            "@scandipwa/m2-theme": true
+            "@scandipwa/m2-theme": true,
+            "@scandipwa/webpack-i18n-runtime": true
```
Comment (alfredsgenkins) [inline packages/scandipwa/package.json:36]: Where is the dependency itself? :)

**Review by alfredsgenkins — CHANGES_REQUESTED** [review]

**Comment by alfredsgenkins** [comment]
Any changes in PHP template? Preloads not ready yet?

File: packages/scandipwa/package.json (hunk 1 above)
Comment (yeegor) [reply packages/scandipwa/package.json]: Is it possible to add the dependency before the package itself is published to the `npm`? 🤔

Comment (alfredsgenkins) [reply packages/scandipwa/package.json]: Just do it!

Comment (yeegor) [reply packages/scandipwa/package.json]: I am surprised not to see package-lock in the project :D will add the dep now.

**Pull request closed without merging**